### PR TITLE
Tatar Translation for Fabric API.

### PR DESCRIPTION
**Pull request by Amirhan-Taipovjan-Greatest-I**
Hello!
There's all Fabric API's translation files that in Tatar language. Can you add them, please?

### CODE REVIEW

**Review by Technici4n — CHANGES_REQUESTED** [review]
Thank you for the contribution. Please remove the translations for the testmods.

**Comment by Amirhan-Taipovjan-Greatest-I** [comment]
> Thank you for the contribution. Please remove the translations for the testmods.

Did I delete all translations for the testmods?

File: fabric-game-rule-api-v1/src/testmod/resources/assets/fabric-gamerule-test/lang/tt_ru.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (Technici4n) [inline fabric-game-rule-api-v1/src/testmod/resources/assets/fabric-gamerule-test/lang/tt_ru.json]: This is still a testmod

Comment (Amirhan-Taipovjan-Greatest-I) [reply fabric-game-rule-api-v1/src/testmod/resources/assets/fabric-gamerule-test/lang/tt_ru.json]: > This is still a testmod

I deleted it too.

**Comment by Amirhan-Taipovjan-Greatest-I** [comment]
I guess... there's no testmod translations now.

**Pull request closed without merging**